+Anim/add_coordinate_sys.m

```diff
@@ -24,4 +24,7 @@ function add_coordinate_sys(rbm)
 
 end
 
+function rng = range(inp)
```

Comment:
@pymBRT It would be good to add the range function in one place and import it in both functions, instead of having two identical copies.